apps/nextra/pages/en/build/indexer/_meta.tsx

```diff
@@ -1,23 +1,30 @@
 export default {
-  "---concepts---": {
+  "---usage---": {
     type: "separator",
-    title: "Concepts",
+    title: "Usage",
+  },
+  "aptos-hosted": {
+    title: "Indexer API Access",
   },
   "indexer-reference": {
     title: "Indexer API Table Reference",
   },
   architecture: {
     title: "Architecture",
   },
```

Comment:
I wouldn't recommend putting architecture at the top because the most important page for users to access it the "Access" page (the current top page, formerly called "Labs-Hosted Indexer API").

I can put it above the table reference, but the reason I put that 2nd was to support the API use case first, then everything else second.

Reply:
Ok fair